cpp/include/cudf/quantiles.hpp

```diff
@@ -42,5 +42,27 @@ quantile(column_view const& input,
          interpolation interp = interpolation::LINEAR,
          order_info column_order = {});
 
+/* @brief Selects the rows corrosponding to a quantiles by interpolating between
```

Comment:
Does this API support if the `input` is already sorted? I didn't mention that in the original requirements, but we definitely need to support this usecase.

Reply:
We can add support for a pre-sorted _table_ pretty easily, but if we need to know pre-sortedness by _column_ (some presorted, some not), then that will need to be implemented in `sorted_order`, which I imagine is a pretty big project.

In the short term, we can either flag the input as pre-sorted, or use `is_sorted` on the table. Preference?

Reply:
Yeah, I was referring to if the _table_ is already sorted according to the per-sorted column/null ordering. I think a simple flag is sufficient.